mod-source-record-manager-server/src/test/java/org/folio/services/JobMonitoringServiceImplTest.java

```diff
@@ -146,4 +158,26 @@ public void shouldDeleteJobMonitoring(TestContext context) {
       async.complete();
     });
   }
+
+  @Test
+  public void shouldFindAllJobMonitoring(TestContext context) {
+    Async async = context.async();
+    JobMonitoring givenJobMonitoring = new JobMonitoring();
```

Comment:
We don't use the 'givenJobMonitoring' in this test, do we need it here ?

Reply:
Nice catch!